### ISSUE TITLE
Update DropBox example to APIv2

### ISSUE THREAD

**Issue by dr-dimitru** [veliovgroup/Meteor-Files#112]
For more see: https://blogs.dropbox.com/developers/2016/06/api-v1-deprecated/

**Comment by dr-dimitru** [comment]
Meanwhile please use: `npm install dropbox-js@=0.10.3`

**Comment by LeG3nDz** [comment]
Hi @dr-dimitru 

I've try to use Dropbox with your update, and it don't work for me. I've add the  cod example to my images.collection.js and after upload, i've not new upload on Dropbox. I've no erreur on my console :

```
I20160712-15:21:50.844(2)? [FilesCollection] [find({}, undefined)]
I20160712-15:21:54.613(2)? [FilesCollection] [File Start Method] Chrysanthemum.jpg - tzjGNJu9FGD4JoKgX
I20160712-15:21:54.618(2)? [FilesCollection] [Upload] [Start Method] Got #-1/3 chunks, dst: Chrysanthemum.jpg
I20160712-15:21:55.276(2)? [FilesCollection] [Upload] [DDP] Got #1/3 chunks, dst: Chrysanthemum.jpg
I20160712-15:21:55.370(2)? [FilesCollection] [Upload] [DDP] Got #3/3 chunks, dst: Chrysanthemum.jpg
I20160712-15:21:55.375(2)? [FilesCollection] [Upload] [DDP] Got #2/3 chunks, dst: Chrysanthemum.jpg
I20160712-15:21:55.447(2)? [FilesCollection] [Upload] [DDP] Got #-1/3 chunks, dst: Chrysanthemum.jpg
I20160712-15:21:55.449(2)? [FilesCollection] [Upload] [finish(ing)Upload] -> assets\app\uploads\Images\tzjGNJu9FGD4JoKgX.jpg
I20160712-15:21:55.451(2)? [FilesCollection] [Upload] [finish(ed)Upload] -> assets\app\uploads\Images\tzjGNJu9FGD4JoKgX.jpg
I20160712-15:21:55.453(2)? [FilesCollection] [_preCollectionCursor.observeChanges] [removed]: tzjGNJu9FGD4JoKgX
```

Do you have any idea ? (my Dropbox access is correct)

Thank you for your help :)

**Comment by dr-dimitru** [comment]
1. Log is looks good to me
2. Could you fork demo app (or use heroku deploy button) with your DropBox credentials to see if it works?
3. Make sure you're using API v1 on DB and your app

**Comment by dr-dimitru** [comment]
@LeG3nDz any news in your end? Have you make it work?

**Comment by dr-dimitru** [comment]
@LeG3nDz any update from your end? Have you figured out how to make it work?

**Comment by LeG3nDz** [comment]
Hi @dr-dimitru,

I have not tried for a few months now. Currently, I use another plugin with Amazon because I could not make it work.

**Comment by Buom01** [comment]
If it can help anybody, this is my code using latest `dropbox@^2.5.1` package:
https://gist.github.com/Buom01/d88243099d27d23c236634a59bda53a1

**Comment by mgscreativa** [comment]
Hi! in this issue entry you have my working code with dropbox v2.5.3 and ostrio:files v1.7.17 https://github.com/VeliovGroup/Meteor-Files/issues/435#issuecomment-305363734

i have used the modules pattern to hide server upload logic fro client for security, but you can implement the whole code in the collection.

**Comment by mibcadet** [comment]
Hi, is it updated for dropbox v2?

**Comment by OngBona** [comment]
<h1>Hello</h1>
<button>Add</button>

**Comment by dr-dimitru** [comment]
@mibcadet @mgscreativa @jankapunkt hope someone can contribute and update our docs [located here](https://github.com/VeliovGroup/Meteor-Files/blob/master/docs/dropbox-integration.md), I'll try to bring more community attention to this issue, thank you for requesting and reminding about this issue

**Comment by RezaRahemtola** [comment]
Hey there,
Since the integration made by @Buom01 is more than 3 years old now, I've faced some issues during its implementation in my project, so if it can help anybody this is my code using the latest `dropbox@^4.0.30` package:
https://gist.github.com/RezaRahemtola/b87cf5f9da0a10f17ec2ca1164a2013b

**Comment by dr-dimitru** [comment]
Awesome, @RezaRahemtola thank you for contribution, do you wish to send a PR?

**Comment by RezaRahemtola** [comment]
@dr-dimitru Just created one :)

**Comment by dr-dimitru** [comment]
Solved in #751 
Thank you @RezaRahemtola
